repl/src/main/scala/ammonite/repl/interp/Compiler.scala

```diff
@@ -101,7 +101,9 @@ object Compiler{
 
       def display(pos: Position, msg: String, severity: Severity) = {
         severity match{
-          case ERROR => logger(Position.formatMessage(pos, msg, false))
+          case ERROR => 
+            println(s"ERROR: $msg")
```

Comment:
Yes... darn! You are right, it shows on every compilation error. Any suggestion as to the best way to fix this?

Editorial comment: I actually prefer the error printed by this line, to the more detailed "Compilation Failed" error the REPL currently prints.
for example, the current REPL:

```
billb-billb @: bad line
Compilation Failed
Main.scala:125: not found: value bad
bad line
^
billb-billb @:
```

* its verbose
* the `Main.scala:125:` is meaningless to the user.

The error printed by the new error println is quite succinct: 

```
billb-Ammonite (master) @: bad line
ERROR: not found: value bad
Compilation Failed
Main.scala:125: not found: value bad
bad line
^
billb-Ammonite (master) @:
```

Still... the new error println is inconsistent and repetitive.